workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -100,7 +109,9 @@ internal static DownloadResult TryDownload()
                 return DownloadResult.AlreadyInstalled;
             }
 
-            return Download();
+            DownloadResult result = Download();
+            SetPluginsCompatibility();
```

Comment:
Given this is called after `Download` in both cases - can this call be moved into the `Download()` function?

Reply:
We have to refresh Asset Database before we can set plugins compatibility. This part actually doesn't work well with generation, as refresh doesn't happen.
Let me move that into Download along with AssetDatabase.Refresh() call